### DISCUSSION TITLE
Read-only Resources based on a custom query

### DISCUSSION THREAD

**Discussion by nirev** [naymspace/backpex#547] [Q&A] [answered]
Hello!

I'm looking into a tool for admin purposes and one of the things we need to do is have some tabular reports that are based on a custom query that looks into multiple tables. That would be read-only, just for showing data that our PO needs.

From what I seen in the docs, the best way to do that would be to create a module with my schema, pointing to the main table I'm selecting, and providing empty changeset functions. Is that the proper way to do that?

**Comment by pehbehbeh** [comment, marked as answer]
Correct, you can just provide empty changeset functions and disable all edit/create/delete stuff via [authorization](https://hexdocs.pm/backpex/live-resource-authorization.html).
